app/app.js

```diff
@@ -28,7 +29,9 @@ const commentRoutes     = require('./routes/comments'),
 
 
 const ONE_KILOBYTE_LIMIT = '1kb'
-const DEV_MODE = process.env?.DEV_MODE ?? true 
+const DEV_MODE = process.env?.DEV_MODE ?? true
+const MAX_FILE_SIZE = process.env.MAX_FILE_SIZE ?? 8 * 1024 * 1024// 8 MB
+const MAX_FILE_COUNT = process.env.MAX_FILE_COUNT ?? 5 //5 Files max per request
```

Comment:
Just 
```
const DEFAULT_MAX_FILE_COUNT = 5
const MAX_FILE_COUNT = process.env.MAX_FILE_COUNT ??  DEFAULT_MAX_FILE_COUNT
```
instead of the comment